### PR TITLE
Update integration.rst

### PR DESCRIPTION
**Pull request by stifoon**

### CODE REVIEW

File: documentation/navitia.io/source/integration.rst [hunk 1]
```diff
@@ -276,6 +276,27 @@ Coverage (/coverage)
...
+
+.. code-block:: json
+
+    {
+        "regions": [
+            "fr-idf"
+        ],
+        address": {
+        ...,
+        "label": "17 Rue Hector Malot (Paris)",
```
Comment (pbougue) [inline documentation/navitia.io/source/integration.rst]: maybe 20?

File: documentation/navitia.io/source/integration.rst [hunk 2]
```diff
@@ -276,6 +276,27 @@ Coverage (/coverage)
 You can easily navigate through regions covered by navitia.io, with the coverage api.
 The only arguments are the ones of `paging`_.
 
+Inverted geocoding (/coord)
+***************************
+In order to determinate the right coverage, you can request navitia with a WGS84 coordinate as:
```
Comment (antoine-de) [inline documentation/navitia.io/source/integration.rst]: I think we should clearly speak about both uses:
- Inverted geocoding (with explaination)
- coverages list

File: documentation/navitia.io/source/integration.rst [hunk 3]
```diff
@@ -276,6 +276,65 @@ Coverage (/coverage)
...
+
+.. code-block:: json
+
+    {
+        "regions": [
+            "fr-idf"
+        ],
+        address": {
+        ...,
+        "label": "20 Rue Hector Malot (Paris)",
```
Comment (TeXitoi) [inline documentation/navitia.io/source/integration.rst:300]: I know @pbougue asked for 20, but the result is not coherent with the request...

Comment (pbougue) [inline documentation/navitia.io/source/integration.rst:300]: The goal is to advertise about Kisio Digital!
First, it's just an example
Second, do we ask datas to be coherent with the truth? ;)

**Pull request merged**